ipspot/ipv4.py

```diff
@@ -174,6 +174,37 @@ def _my_ip_io_ipv4(geo: bool=False, timeout: Union[float, Tuple[float, float]]
         return {"status": False, "error": str(e)}
 
 
+def _ifconfig_co_ipv4(geo: bool=False, timeout: Union[float, Tuple[float, float]]
+                      =5) -> Dict[str, Union[bool, Dict[str, Union[str, float]], str]]:
+    """
+    Get public IP and geolocation using ifconfig.co.
+    
+    :param geo: geolocation flag
+    :param timeout: timeout value for API
+    """
+    try:
+        with requests.Session() as session:
+            response = session.get("https://ifconfig.co/json", headers=REQUEST_HEADERS, timeout=timeout)
```

Comment:
I tested this API, it's not **only-IPv4**

Reply:
ok, I will change it then. Thanks for noticing.